env.dist

```diff
@@ -24,6 +24,9 @@ export S3_EMULATION=true
 # Endpoint for published projects
 export PUBLIC_PROJECT_ENDPOINT="localhost:8001"
 
+# Endpoint for the remix script
```

Comment:
Can you add some extra comments here explaining that the script is responsible for pulling in any resources needed for the remix interface?